src/components/Faq/Faq.tsx

```diff
@@ -0,0 +1,246 @@
+              Decentraland with your email or a social account, a digital wallet
+              will be automatically don&apos;t already have one, when you sign
+              up for Decentraland with your email or a social account, a digital
+              wallet will be automatically created for you. It&apos;s part of
+              your account and stores any assets you collect, like free
+              Wearables you might claim in-world or a fun Emote you purchase in
+              the Marketplace. purchase in the Marketplace.
+            </p>
+          ),
+        },
+        {
+          question: "How do I keep up to date with Music Festival News?",
+          answer: () => (
+            <p>
+              Follow{" "}
+              <a
+                href="https://twitter.com/decentraland"
+                target="_blank"
+                rel="noreferrer"
+              >
+                @Decentraland
+              </a>{" "}
+              on Twitter and keep an eye on the{" "}
+              <a
+                href="https://twitter.com/search?q=%23DCLMF24"
+                target="_blank"
+                rel="noreferrer"
+              >
+                #DCLMF24
+              </a>{" "}
+              hashtag for live updates. You can also subscribe to
+              Decentraland&apos;s{" "}
+              <a
+                href="https://decentraland.beehiiv.com/?utm_org=dcl&utm_source=decentraland&utm_medium=musiclanding&utm_campaign=musicfestival24"
+                target="_blank"
+                rel="noreferrer"
+              >
+                weekly newsletter
+              </a>{" "}
+              for all of the latest news.
+            </p>
+          ),
+        },
+        {
+          question: "I want to create in Decentraland, where can I learn more?",
+          answer: () => (
+            <p>
+              Anyone can become a Decentraland creator, all it takes is a little
+              knowhow and endless creative ideas! Everything you need to know to
+              start your creative journey can be found at{" "}
+              <a
+                href="https://decentraland.org/create/?utm_org=dcl&utm_source=decentraland&utm_medium=musiclanding&utm_campaign=musicfestival24"
+                target="_blank"
+                rel="noreferrer"
+              >
+                decentraland.org/create
+              </a>
+              .
+            </p>
+          ),
+        },
+      ].map((item: Question, index: number) => (
+        <QuestionContainer key={index}>
+          <Question onClick={() => toggleAnswer(index)}>
+            <p>{item.question}</p>
+            <Arrow
+              src={arrowDown}
+              alt="arrow"
+              $isActive={activeIndex === index}
+            />
+          </Question>
+          <Answer $isActive={activeIndex === index}>{item.answer()}</Answer>
+        </QuestionContainer>
+      ))}
+    </FaqContainer>
+  )
+}
+
+const FaqContainer = styled.div`
```

Comment:
np: what do you think about moving these styled components to a `Faq.styled.ts` file?